src/github/orbit-helpers.js

```diff
@@ -323,3 +321,36 @@ export function _getRepositoryFullName() {
     window.location.pathname.split("/")[2]
   }`;
 }
+
+/**
+ * Fetches chunked repositories from chrome storage
+ *
+ * @returns Array<String> a 1d array of all repsoitory names, ie ["repo-1", "repo-2"]
+ */
+export async function _fetchRepositories() {
+  const { repository_keys } = await chrome.storage.sync.get("repository_keys");
+
+  // Backwards compatibility - if we do not have repository keys,
+  //  default to how we used to store them
+  if (repository_keys === undefined) {
+    const { repositories } = await chrome.storage.sync.get("repositories");
```

Comment:
Would it make sense to specify an empty array as the default value here?

Reply:
😰 Okay I'm inclined to leave this as-is. Struggling to switch to it nicely, & I don't think it's worth the time since all we'll be saving is changing a check from `=== undefined` to `.length == 0`. Bigger fish :P